.github/workflows/continuous-integration.yml

```diff
@@ -144,7 +144,7 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: "us-gov-west-1"
+          aws-region: 'us-gov-west-1'
```

Comment:
Actually might not need any quotes at all.

Reply:
removed these quotes entirely